### DISCUSSION TITLE
Proposal: add back Samba for macOS

### DISCUSSION THREAD

**Discussion by AkihiroSuda** [Homebrew/discussions#1736] [Writing Formulae/Casks] [answered]
Samba (v3) was removed from Homebrew in 2016 due to technical issues: https://github.com/Homebrew/legacy-homebrew/issues/17820 https://github.com/Homebrew/homebrew-core/commit/5af93aec46c7cd43b068c9c93eb292ffd1e07fd0

However, Samba v4 seems actively supporting macOS recently.

Actually, MacPorts now supports very recent version of Samba (v4.13.1, released on October 29, 2020) with just a couple of patches: https://github.com/macports/macports-ports/tree/84d5040b7739311e75879d3afb4c4e1f464469d1/net/samba4

And the patch seems no longer needed for Samba v4.14.x:  https://github.com/samba-team/samba/commit/53a1d034f3e47ed3c5ec4f9b6ca0d2d695e8d873

Can we bring back Samba to Homebrew too?

**Comment by SMillerDev** [comment]
Go ahead: https://docs.brew.sh/Adding-Software-to-Homebrew

**Reply by AkihiroSuda** [reply]
Thanks, opened PR https://github.com/Homebrew/homebrew-core/pull/80171

**Comment by AkihiroSuda** [comment, marked as answer]
Now available again

https://github.com/Homebrew/homebrew-core/blob/master/Formula/samba.rb
